app/src/main/java/org/mozilla/focus/home/HomeFragment.kt

```diff
@@ -214,6 +222,17 @@ private class HomeTileAdapter(
         }
     }
 
+    fun setLayoutParams(iconView: View, tileMarginValue: Int, tileHeight: Int) {
+        val marginLayoutParams = iconView.layoutParams as ViewGroup.MarginLayoutParams
+        val marginValue = iconView.resources.getDimensionPixelSize(tileMarginValue)
+        marginLayoutParams.setMargins(marginValue, marginValue, marginValue, marginValue)
+        val layoutParams = iconView.layoutParams as LinearLayout.LayoutParams
```

Comment:
nit: you should be able to set the margins on this reference too (i.e. don't also get `marginLayoutParams`